File: Ruby/7 KYU/Complementary DNA.rb
def DNA_strand(dna)
  new = []
  dna = dna.split("")
  dna.each do |n|
    if n == "A"
      new << "T"
    elsif n == "T"
      new << "A"
    elsif n == "G"
      new << "C"
    elsif n == "C"
      new << "G"
    end
  end
  new.join
end

File: Javascript/7 KYU/Complementary DNA.js
function DNAStrand(dna){
    let letters = '';
    for(let i = 0; i < dna.length; i++){
        switch (dna[i]) {
            case 'T':
                letters += 'A';
                break;
            case 'A':
                letters += 'T';
                break;
            case 'C':
                letters += 'G';
                break;
            case 'G':
                letters += 'C';
                break;
        }
    }
    return letters;
}
